src/theme/typography/theme-helper.ts

```diff
@@ -0,0 +1,95 @@
+import { CSSProp, css } from "styled-components";
```

Comment:
remove unused import

```suggestion
import { CSSProp } from "styled-components";
```